cpp/modules/deck.gl/core/src/arrow/row.cc

```diff
@@ -391,7 +391,7 @@ auto Row::_vector2FromDoubleArray(const std::shared_ptr<arrow::DoubleArray>& val
     return defaultValue;
   }
 
-  // Needed for some reason as seen in https://arrow.apache.org/docs/cpp/examples/row_columnar_conversion.html
+  // Buffer at index 0 contains null bitmap, the actual data is in the buffer at index 1 for majority of the data types
```

Comment:
Assuming this is true even if the type is not nullable (does not have a null bitmap)? Is the first buffer just null in that case?

Should we define constants and put the comment there?

NULL_BITMAP_BUFFER_INDEX...
DATA_BUFFER_INDEX...